tests/Cache/FileHandlerTest.php

```diff
@@ -229,4 +188,21 @@ private function getFile()
     {
         return __DIR__.'/.php_cs.cache';
     }
+
+    /**
+     * @return SignatureInterface
+     */
+    private function getSignature()
```

Comment:
What do you think about using `create` instead of `get` as prefix, to indicate that we aren't retrieving the same instance, but creating a new instance every time the method is invoked?

Reply:
Makes sense, so I did it 😄